cl8-web/src/main.js

```diff
@@ -31,10 +31,9 @@ const dict = {
   }
 };
 
-
 Vue.use(VeeValidate, vvConfig)
 Vue.use(VueFuse)
-Vue.use(VueFire)
+Vue.use(firestorePlugin)
```

Comment:
@rmparr can you let me know if this works in the UI for you when you make this change?

I know that firebase switched from having the realtime database as the default datastore, to using firestore instead, when I wrote this I think all the code assumed use of the realtime database.

If it doesn't, we may need to explicitly use the realtime database for this instead until we make decision about changing it.

Reply:
Yes, it still works as before. Fetching and updating profiles works with this.